backend/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:18-alpine
+WORKDIR /app/backend
+COPY /package*.json ./
+RUN npm install
+COPY . ./
+EXPOSE 8000
```

Comment:
When you have `.dockerignore` file you don't have to use copy method twice. Dockerfile will define files to exclude. So copy onece and then after `npm install` you will have all required dependencies to start an application

Reply:
Corrected, now both files copy only once